python/fledge/plugins/north/opcuaclient/opcuaclient.py

```diff
@@ -73,86 +69,64 @@
          "description": "Source of data to be sent on the stream. May be either readings or statistics.",
          "type": "enumeration",
          "default": "readings",
-         "options": [ "readings", "statistics" ],
+         "options": ["readings", "statistics"],
          'order': '3',
          'displayName': 'Source'
-    },
-    "applyFilter": {
-        "description": "Should filter be applied before processing data",
-        "type": "boolean",
-        "default": "false",
-        'order': '4',
-        'displayName': 'Apply Filter'
-    },
-    "filterRule": {
-        "description": "JQ formatted filter to apply (only applicable if applyFilter is True)",
-        "type": "string",
-        "default": ".[]",
-        'order': '5',
-        'displayName': 'Filter Rule',
-        "validity": "applyFilter == \"true\""
     }
 }
 
+
 def plugin_info():
     return {
-        'name': 'opcuaclient',
-        'version': '1.9.1',
+        'name': 'OPCUA Client',
+        'version': '2.0.1',
         'type': 'north',
         'interface': '1.0',
         'config': _DEFAULT_CONFIG
     }
 
+
 def plugin_init(data):
-    _LOGGER.info('Initializing OpcuaClient North Python Plugin')
     global opcuaclient_north, config
     opcuaclient_north = OpcuaClientNorthPlugin()
     config = data
-    _LOGGER.info(f'Initializing plugin with Primary Connection String: {config["url"]["value"]}')
     return config
 
+
 async def plugin_send(data, payload, stream_id):
     try:
-        _LOGGER.info(f'OpcuaClient North Python - plugin_send: {stream_id}')
         is_data_sent, new_last_object_id, num_sent = await opcuaclient_north.send_payloads(payload)
     except asyncio.CancelledError as ex:
         _LOGGER.exception(f'Exception occurred in plugin_send: {ex}')
     else:
-        _LOGGER.info('payload sent successfully')
+        _LOGGER.debug('payload sent successfully')
         return is_data_sent, new_last_object_id, num_sent
 
+
 def plugin_shutdown(data):
     pass
 
-# TODO: North plugin can not be reconfigured? (per callback mechanism)
+
 def plugin_reconfigure():
     pass
 
+
 class OpcuaClientNorthPlugin(object):
     """ North Opcua Client Plugin """
 
     def __init__(self):
         self.event_loop = asyncio.get_event_loop()
 
-    def opcuaclient_error(self, error):
-        _LOGGER.error(f'OpcuaClient error: {error}')
-
     async def send_payloads(self, payloads):
         is_data_sent = False
         last_object_id = 0
         num_sent = 0
 
-        size_payload_block = 0
-
-        map = json.loads(config['map']['value'])
-
         try:
-            _LOGGER.info('processing payloads')
-            payload_block = list()
-
+            _LOGGER.debug('processing payloads')
```

Comment:
It would be helpful to include the size of the payloads collection in this message.